### PR TITLE
[UCSC-170] Photo of the Week archive

### PR DESCRIPTION
**Pull request by bjcooper**
## What does this do/fix?

Added FE for the Photo of the Week archive. In so doing:
- Added query loop pagination styles
- Added SCSS mixins, including importing the theme's mixins for use in the plugin
- Added templating for the archive route and CPT loop block
- Added custom styles

### Links to relevant issues

- https://moderntribe.atlassian.net/browse/UCSC-170

### CODE REVIEW

File: src/views/photos_week_loop/index.php [hunk 1]
```diff
@@ -10,31 +10,51 @@
...
+			</figure>
 			<?php endwhile; ?>
-			</ul>
+	
 			<!-- Pagination -->
 			<nav class="wp-block-query-pagination is-content-justification-center is-layout-flex wp-container-core-query-pagination-is-layout-1 wp-block-query-pagination-is-layout-flex">
-				<?php
-				echo $c->get_pagination( $posts, $paged );
-				?>
+				<?php if ( $paged === 1 ) : ?>
```
Comment (MlKilderkin) [inline src/views/photos_week_loop/index.php:45]: I believe we should not show buttons if we don't have other pages. So you may want to adjust your statement to the 
`<?php if ( $paged === 1 && $posts->max_num_pages > 1 ) : ?>` 
Same will applies to next button
`<?php if ( $paged === $posts->max_num_pages && $posts->max_num_pages > 1 ) : ?>`

**Review by MlKilderkin — COMMENTED** [review]
🟡

File: src/views/photos_week_loop/index.php (hunk 1 above)
Comment (bjcooper) [reply src/views/photos_week_loop/index.php:45]: The design calls for the previous and next buttons to always be present in the pager, but disabled when they don't do anything. As I understand it, the `get_pagination()` logic leaves off the previous button on the first page and leaves off the next button on the last page, so those are the cases I tried to cover here.

Comment (MlKilderkin) [reply src/views/photos_week_loop/index.php:45]: Page looks a bit odd when we have 2 disabled buttons and only 1 page. get_pagination will hide any buttons if there is only 1 page
Leave it as is but I'd clarify this with design and strategy

**Review by MlKilderkin — APPROVED** [review]

File: src/views/photos_week_loop/index.php (hunk 1 above)
Comment (bjcooper) [reply src/views/photos_week_loop/index.php:45]: Agree that it looks odd, but it is what's shown in the design so I was gonna let  design bring it up in DQA. I'll call it out specifically in the testing instructions, Thank you!

**Pull request merged**